packages/@react-spectrum/dialog/src/DialogTrigger.tsx

```diff
@@ -144,13 +144,48 @@ export {_DialogTrigger as DialogTrigger};
 
 function PopoverTrigger({state, targetRef, trigger, content, hideArrow, isKeyboardDismissDisabled, ...props}) {
   let triggerRef = useRef<HTMLElement>();
-
   let overlayRef = useRef<DOMRefValue<HTMLDivElement>>();
+  let isCloserPadding = false;
```

Comment:
what is isCloserPadding?

does it mean `isNextToContainerBoundary`? or `shouldUseLessPadding`?